.github/workflows/release.yml

```diff
@@ -8,37 +8,8 @@ on:
 
 jobs:
   release:
-    name: Create Release
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-        python-version: [3.9]
-      fail-fast: false
-    steps:
-      - name: Check out source code
-        uses: actions/checkout@v2
-      - name: Create Release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          python-version: ${{ matrix.python-version }}
-          tag_name: ${{ github.ref }}
-          release_name: Release ${{ github.ref }}
-          body: |
-            Changes in this Release
-          draft: false
-          prerelease: false
-
-  deploy:
-    needs: release
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-        python-version: [3.9]
-      fail-fast: false
+    runs-on: ubuntu-latest
+    python-version: 3.9
```

Comment:
The `python-version: 3.9` goes inside the "Set up Python" step (replacing the `python-version: ${{ matrix.python-version }}`).

Reply:
oops, thanks!